src/gui/creds/flow2auth.cpp

```diff
@@ -277,11 +220,58 @@ void Flow2Auth::slotPollTimerTimeout()
     });
 }
 
+QJsonObject Flow2Auth::handleRequest(QNetworkReply *reply)
```

Comment:
nit: would rename this method to `handleResponse` as it's dealing with the received reply instead of the sent request